tests/test_mapped_estimators.py

```diff
@@ -100,8 +100,10 @@ def test_two_sided_estimates():
         w_F = compute_mapped_reduced_work(x_a, u_a, u_b, map_fxn)
         w_R = compute_mapped_reduced_work(x_b, u_b, u_a, inv_map_fxn)
 
-        # estimated_delta_f = BAR(w_F, w_R)[0] #  default solver -> BoundsError: Cannot determine bound on free energy
-        estimated_delta_f = BAR(w_F, w_R, method="self-consistent-iteration", compute_uncertainty=False)
+        # estimated_delta_f = pymbar.bar(w_F, w_R)["Delta_f"] #  default solver -> BoundsError: Cannot determine bound on free energy
```

Comment:
Random: this comment was based on pymbar 3, unsure if still relevant with pymbar 4's default solver

Reply:
It is still relevant. Uncommenting the commented line results in the same `BoundsError` under pymbar4.

Reply:
Thanks for confirming!